src/components/Body.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import Facets from "../containers/Facets";
+import PagingInfo from "../containers/PagingInfo";
+import Paging from "../containers/Paging";
+import Results from "../containers/Results";
+import ResultsPerPage from "../containers/ResultsPerPage";
+import Sorting from "../containers/Sorting";
```

Comment:
As with above, is there any reason for this over a single index.js file for all of these?
```
import { 
  Facets,
  PagingInfo,
  Paging,
  Results,
  ResultsPerPage,
  Sorting
} from "../containers";
```